src/Functions/materialize.h

```diff
@@ -23,6 +23,11 @@ class FunctionMaterialize : public IFunction
         return false;
     }
 
+    bool useDefaultImplementationForNothing() const override
```

Comment:
why?

Reply:
Actually after adding throwing exception on creating non-empty Nothing column this is not needed, removed